source/installation/install-on-windows.txt

```diff
@@ -0,0 +1,99 @@
+.. _c2c-install-windows:
+
+==================
+Install on Windows
+==================
+
+.. include:: /includes/preview-warning
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+.. |arrow| unicode:: U+27A4
+
+Overview
+--------
+
+Use this tutorial to manually install {+c2c-full-product-name+}
+{+version+} on Windows using a downloaded ``.zip`` zipfile.
+
+.. bug in constant parser DOP-3025
+
+{+c2c-product-name+} Version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/fact-install-past-mongosync.rst
+
+Considerations
+--------------
+
+Consider the following when running {+c2c-full-product-name+} on Windows:
+
+Platform Support
+~~~~~~~~~~~~~~~~
+
+{+c2c-product-name+} is available on the following Windows operating
+systems:
+
+- Windows 10
+- Windows 11
+
+Version Requirements
+~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/fact-mongodb-version.rst
+
+.. bug in constant parser DOP-3025
+
+Install {+c2c-product-name+}
+-------------------------------
+
+Follow these steps to manually install {+c2c-product-name+} from
+the ``.zip``.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Download the zipfile.
+
+      Download the {+c2c-product-name+} zipfile from the following
+      link:
+
+      |arrow| {+mdb-download-center+}
```

Comment:
[Question]

Will this link need to be updated tomorrow? Right now, I assume it doesn't work for good reason.